src/pages/contact/index.tsx

```diff
@@ -0,0 +1,298 @@
+    padding: 0,
+    [theme.breakpoints.up('lg')]: {
+      display: 'flex',
+      padding: theme.spacing(8, 2, 0, 2),
+    },
+  },
+
+  main: {
+    [theme.breakpoints.up('lg')]: {
+      width: '100%',
+    },
+  },
+
+  detailsCard: {
+    margin: theme.spacing(0, 2.5, 0, 2.5),
+    [theme.breakpoints.up('lg')]: {
+      height: 668,
+      width: 280,
+      display: 'block',
+      position: 'sticky',
+      marginRight: theme.spacing(4),
+    },
+  },
+
+  mainContent: {
+    backgroundColor: theme.palette.background.paper,
+    borderRadius: 16,
+    padding: theme.spacing(3),
+    [theme.breakpoints.down('lg')]: {
+      margin: theme.spacing(4, 2.5, 4, 2.5),
+    },
+    [theme.breakpoints.down('xs')]: {
+      margin: theme.spacing(4, 0, 0, 0),
+    },
+
+    [theme.breakpoints.up('lg')]: {
+      display: 'flex',
+      flexDirection: 'column',
+      margin: theme.spacing(0, 0, 7, 0),
+      boxShadow: contactPageItemShadow,
+      width: '100%',
+      padding: theme.spacing(4),
+    },
+  },
+
+  navbar: {
+    marginBottom: theme.spacing(2.5),
+    [theme.breakpoints.up('sm')]: {
+      marginBottom: theme.spacing(3),
+    },
+    [theme.breakpoints.up('lg')]: {
+      marginBottom: theme.spacing(7),
+    },
+  },
+
+  inputs: {
+    display: 'grid',
+    gridTemplateRows: '1fr',
+    gridRowGap: theme.spacing(4),
+    [theme.breakpoints.up('sm')]: {
+      gridRowGap: theme.spacing(4),
+    },
+    [theme.breakpoints.up('lg')]: {
+      gridTemplateRows: '1fr',
+      gridGap: theme.spacing(4),
+    },
+  },
+
+  nameAndEmail: {
+    display: 'grid',
+    gridGap: theme.spacing(3),
+    '&.MuiFormControl-fullWidth': {
+      margin: theme.spacing(0, 2, 4, 2),
+    },
+    [theme.breakpoints.up('sm')]: {
+      justifyContent: 'space-between',
+      gridTemplateColumns: '1fr 1fr',
+      gridGap: theme.spacing(4),
+    },
+    [theme.breakpoints.up('lg')]: {
+      gridTemplateColumns: '1fr 1fr',
+      gridGap: theme.spacing(5),
+      justifyContent: 'space-between',
+    },
+  },
+
+  buttonWrapper: {
+    display: 'flex',
+    width: '100%',
+    justifyContent: 'flex-end',
+    paddingTop: theme.spacing(4),
+  },
+
+  button: {
+    width: 158,
+    height: 40,
+    borderRadius: 8,
+    boxShadow: '0 2px 4px 0 rgba(47, 84, 235, 0.15), 0 8px 16px 0 rgba(47, 84, 235, 0.15)',
+  },
+
+  title: {
+    marginBottom: theme.spacing(4),
+    [theme.breakpoints.up('sm')]: {
+      marginBottom: theme.spacing(3),
+    },
+    [theme.breakpoints.up('lg')]: {
+      margin: theme.spacing(0, 2, 4, 2),
+    },
+  },
+}));
+
+const ContactPage = () => {
+  const classes = useStyles();
+
+  const developerProfile = useDeveloperProfile();
+  const { componentType, isDesktop, isMobile, isTablet } = useComponentType();
+
+  return (
+    <Container className={classes.container} maxWidth="lg">
+      <Helmet>
+        <title>Contact page</title>
+      </Helmet>
+      {isDesktop && (
+        <Box className={classes.detailsCard}>
```

Comment:
This card should be defined as a separate component with all its props

Reply:
I created new issues